### ISSUE TITLE
Move exported headers into `include/` dirs

### ISSUE THREAD

**Issue by pb-cdunn** [PacificBiosciences/blasr_libcpp#91]
(This will wait until have the next smrt-release. Just recording some discussion.)

See how complicated header installation is?
* https://github.com/PacificBiosciences/pith/blob/master/sh/mobs/pacbio/libblasr/buildctl.sh#L231

And in mobs, we are not even consistent:
```
deplinks/libblasr/include:
alignment

deplinks/libpbdata/include:
alignment                  CompressedDNASequence.hpp   FASTAReader.hpp    loadpulses         PacBioDefs.h              sam               utils
amos                       CompressedSequence.hpp      FASTASequence.hpp  matrix             PackedDNASequence.hpp     SeqUtils.hpp      utils.hpp
CCSSequence.hpp            CompressedSequenceImpl.hpp  FASTQReader.hpp    MD5Utils.hpp       qvs                       SeqUtilsImpl.hpp  utilsImpl.hpp
ChangeListID.hpp           defs.h                      FASTQSequence.hpp  MD5UtilsImpl.hpp   reads                     SMRTSequence.hpp  VectorUtils.hpp
CommandLineParser.hpp      DNASequence.hpp             GFFFile.hpp        metagenome         ReverseCompressIndex.hpp  StringUtils.hpp
Compare4BitCompressed.hpp  Enumerations.h              libconfig.h        NucConversion.hpp  saf                       Types.h

deplinks/libpbihdf/include:
hdf
```
And both pbdata and alignment export `alignment/` directories, which overlap.

It would be much easier if exported headers were under `include/` directories:
```
blasr_libcpp/alignment/include/blasr/*/*
blasr_libcpp/pbdata/include/pbdata/*/*
blasr_libcpp/hdf/include/pbihdf/*/*
```

We would also have to change blasr, pbdagcon, bam2bax, and bax2bam to include these. It's work, but everything would be *much* easier to understand and to work with.

**Comment by pb-cdunn** [comment]
@ylipacbio says:
> Let's do it after this release

**Comment by pb-cdunn** [comment]
Addressed by #115 (and preceding).
